### ISSUE TITLE
Update CreateTrainJob.m with guess of what iteration of trained model to use

### ISSUE THREAD

**Issue by coleslaw481** [CRBS/cdeep3m#5]
Add this either end of CreateTrainJob.m, but this code should look at out.log for each model and get the loss value for all iterations and find where the slope of this curve levels out or goes below some threshold. Output this value into a README.txt file that is put in the output of the train directory. 

```Bash
grep "loss = " LOG/* | sed "s/^.*]//"
```
